scripts/bootstrap-aio.yml

```diff
@@ -36,6 +36,28 @@
 - name: Execute the OSA AIO bootstrap
   include: "{{ lookup('env', 'OA_DIR') }}/tests/bootstrap-aio.yml"
   vars:
```

Comment:
Hmm..It seems as if the bootstrap role still uses the values specified in the OSA bootstrap-aio.yml playbook.
https://github.com/openstack/openstack-ansible/blob/stable/newton/tests/bootstrap-aio.yml#L25
@odyssey4me Where else could I specify these overrides so they take precedence?